Modules/IO/GE/src/itkGE4ImageIO.cxx

```diff
@@ -191,7 +191,7 @@ GE4ImageIO::ReadHeader(const char * FileNameToRead)
     hdr->coordinateOrientation =
       itk::SpatialOrientationEnums::ValidCoordinateOrientations::ITK_COORDINATE_ORIENTATION_RSP;
   }
-  // RGEDEBUG(std::sprintf (debugbuf, "Plane = %d\n", hdr->imagePlane); cerr <<
+  // RGEDEBUG(std::snprintf (debugbuf, sizeof(debugbuf), "Plane = %d\n", hdr->imagePlane); cerr <<
   // debugbuf;)
```

Comment:
I very much appreciate that you are chasing those unsafe `sprintf` calls. However, if this code was actually being used, I think it would be preferable to insert the values directly into the output stream: 
```cpp
    std::cerr << "Plane = " << hdr->imagePlane << '\n';
```
Right? Anyway, that's probably beyond the scope of this PR!